src/column.js

```diff
@@ -135,9 +135,14 @@ function writeDictionaryPage(writer, column, dictionary) {
  * @import {ColumnMetaData, DecodedArray, Encoding, ParquetType, SchemaElement, Statistics} from 'hyparquet'
  * @import {ColumnEncoder, ListValues, Writer} from '../src/types.js'
  * @param {DecodedArray} values
- * @returns {Statistics}
+ * @param {SchemaElement} element
+ * @returns {Statistics | undefined}
  */
-function getStatistics(values) {
+function getStatistics(values, element) {
+  const ltype = element?.logical_type?.type
+  const isGeospatial = ltype === 'GEOMETRY' || ltype === 'GEOGRAPHY'
+  if (isGeospatial) return
```

Comment:
I think that only min/max statistics must be avoided, but null count is expected for geospatial columns as well.

Also, in a future PR, we could compute GeospatialStatistics (which is in a different ColumnMetaData field)